R/generate_simulated_data.R

```diff
@@ -291,6 +340,47 @@ generate_simulated_data <- function(r_in_weeks = # nolint
     )
   }
 
+  log_r_site <- spatial_rt_process(
+    log_r_state_week,
+    corr_function,
+    corr_fun_params,
+    phi_rt,
+    sigma_eps
+  )
+  # Auxiliary Site
+  if (aux_site_bool) {
```

Comment:
Just for my own clarity, if `use_spatial_corr = FALSE` whether or not the user specifies `aux_site_bool` as true or false doesn't matter, because it will get reassigned as FALSE, and the `independenc_corr_fun()` will spit out a diagnonal matrix of of `n_sites + 1` rows ?

Reply:
Yes, but this can be changed.  I think it actually might be better to allow `aux_site_bool` to be true if `use_spatial_corr` is false.